core/services/synchronization/web_socket_client.go

```diff
@@ -64,6 +81,16 @@ func NewWebSocketClient(url *url.URL, accessKey, secret string) WebSocketClient
 	}
 }
 
+// Url dereferences the URL the client was initialized with
+func (w *websocketClient) Url() url.URL {
+	return *w.url
+}
+
+// Status returns the private attribute
+func (w *websocketClient) Status() ConnectionStatus {
+	return w.status
+}
+
```

Comment:
"// Url returns the URL the client was initialized with", "// Status returns the connection status"?  Users may be interested more in 'what' the functions are doing instead of the 'how'.